### DISCUSSION TITLE
Unable to open :(

### DISCUSSION THREAD

**Discussion by twistedpro** [antonreshetov/timefall#4] [Q&A]
I have tried downloading with both Safari and Chrome, however after extracting the app, I'm unable to open it, the OS suggests it's damaged.

![image](https://github.com/antonreshetov/timefall/assets/20642311/400ec996-c542-4852-b5e1-1756e8d43a8c)

**Comment by antonreshetov** [comment]
@twistedpro try to

1. Go to System Preference
2. Click Security & Privacy
3. Select the Allow button under the Allow apps downloaded from section

This is one of the standard messages for unsigned applications on macOS.
